config/libraries.yaml

```diff
@@ -2,13 +2,13 @@ libraries:
   - name: copy-offload
     lib:
       name: libcopyoffload.a
-      dest: /opt/local/lib
+      dest: /opt/local/lib/
     header:
```

Comment:
We can remove 'header' from here, and from config/config.go.

Reply:
Done.